transformer.js

```diff
@@ -0,0 +1,45 @@
+const transformCart = (cart) => {
+  const cartObj = cart.toObject();
+
+  cartObj.cartLines = cartObj.cartLines.map(cartLine => ({
+    ...cartLine,
+    _id: undefined,
+    product: {...cartLine.product, id: cartLine.product._id, _id: undefined}
+  }));
+  cartObj._id = undefined;
+  cartObj.uid = undefined;
+  cartObj.__v = undefined;
```

Comment:
this code could be extracted to function to avoid DRY code

Reply:
This has been moved to a function